index.bs

```diff
@@ -630,9 +673,11 @@ run these steps:
 		of the {{document}}, and update |container| to be
 		the <a>browsing context container</a> of |container|.
 	2. Map |intersectionRect| to the coordinate space of |container|.
-	3. If |container| has a <a>content clip</a> or a css <a>clip-path</a> property,
+	3. If |container| is scrollable, apply the {{IntersectionObserver}}’s
```

Comment:
Would be great to have a more precise definition of `scrollable`, or a link to the relevant CSS spec. Otherwise whether this applies to `overflow: hidden` or not is a bit questionable, right?

Reply:
Good point. How about using CSS [scroll-container](https://www.w3.org/TR/css-overflow/#scroll-container)?

Reply:
Sounds good, let's make sure that we test overflow: hidden then, and confirm that it does apply.

Reply:
Thanks Emilio!
The tests I added in my [PR for the wpt tests](https://chromium-review.googlesource.com/c/chromium/src/+/4740913) use `overflow: hidden`. For example the [scroll-margin.html](https://chromium-review.googlesource.com/c/chromium/src/+/4740913/9/third_party/blink/web_tests/external/wpt/intersection-observer/scroll-margin.html) test.
I confirmed these pass with my [prototype](https://chromium-review.googlesource.com/c/chromium/src/+/4740520).